resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -381,4 +381,42 @@ object KubernetesUtils extends Logging {
       }
     }
   }
+
+  /**
+   * This function builds the EnvVar objects for each key-value env.
+   */
+  @Since("3.4.0")
+  def buildEnvVarsWithKV(env: Map[String, String]): Seq[EnvVar] = {
```

Comment:
```suggestion
  def buildEnvVars(env: Map[String, String]): Seq[EnvVar] = {
```
IMO there is no need of `WithKV` - the passed argument is not a Seq of tuples anymore.

Reply:
Thanks for your comment :) Useful for me